traitsui/api.py

```diff
@@ -159,6 +159,14 @@
 - :class:`~.RuleTableFilter`
 - :class:`~.TableFilter`
 
+Instance UI
+-----------
+
+- :class:`~.InstanceChoice`
+- :class:`~.InstanceChoiceItem`
+- :class:`~.InstanceDropChoice`
+- :class:`~.InstanceFactoryChoice`
+
```

Comment:
Let's move the `InstanceEditor` down here, it'll be cleaner.

Reply:
There are comparable sections for "Tree UI" and "Table UI", but the `TreeEditor` and `TableEditor` are still included in the full list with the other editors not in those sections.  I am still happy to make this suggested change, but wanted to point this out from a consistency stand point

Reply:
In that case, let's move the `TreeEditor` and `TableEditor` to the Tree UI and Table UI sections respectively - along with the change I recommended earlier.

Alternatively, if we want to list all of the available editors in one location i.e. under the Editors section (how I think things are currently), we can choose to add duplicate information in the Table UI, Tree UI and Instance UI sections.